woocommerce/storefront
# Remove skip-link-focus-fix.js

skip-link-focus-fix is only applicable in IE11, but it gets loaded even on the +95% of browsers which will never use it.

Now that [Matt is supportive of dropping IE11 support in WP Core](https://make.wordpress.org/core/2021/03/04/discussion-dropping-support-for-ie11/#comment-40870), I think it is the best time to remove it.

### How to test the changes in this Pull Request:
1. Open any StoreFront page in Safari (which is the only major browser using webkit).
2. Tab to skip link.
3. Press Enter to activate it.
4. Make sure keyboard focus has moved to the first [focusable area](https://html.spec.whatwg.org/multipage/interaction.html#focusable-area).
5. Repeat this process for Opera and the other major browsers.

### Changelog

> Enhancement – Remove skip-link-focus-fix.js. https://github.com/woocommerce/storefront/pull/1653
